scripts/Deepcell_Preprocessing.ipynb

```diff
@@ -45,15 +45,29 @@
     "# load TIFs from GUI-based directory structure\n",
     "data_dir = \"../data/example_dataset/input_data/\"\n",
     "\n",
+    "# data format\n",
```

Comment:
Might be helpful to include a comment to explain when to set MIBItiff to True for those with a non-technical background, similar to legacy_deepcell in Deepcell_Postprocessing.ipynb.